### PR TITLE
add grpc api for exec

### PR DESCRIPTION
**Pull request by Crazykev**
part of #278 

Signed-off-by: Crazykev <crazykev@zju.edu.cn>

### CODE REVIEW

File: integration/client.go [hunk 1]
```diff
@@ -214,3 +215,72 @@ func (c *HyperClient) RemovePod(podID string) error {
 
 	return nil
 }
+
+// ContainerExec exec a command in a container with input stream in and output stream out
+func (c *HyperClient) ContainerExec(container, tag string, command []byte, tty bool, stdin io.ReadCloser, stdout, stderr io.Writer) error {
```
Comment (feiskyer) [inline integration/client.go]: Just `command string` here, avoid unnecessary type conversions.

File: integration/client.go [hunk 2]
```diff
@@ -214,3 +215,72 @@ func (c *HyperClient) RemovePod(podID string) error {
...
+	request := types.ContainerExecRequest{
+		Type:    "container",
+		Value:   container,
+		Command: string(command),
```
Comment (feiskyer) [inline integration/client.go]: Command should be json marshaled:

```go
cmd, err := json.Marshal(command)
    if err != nil {
        return err
    }
```

And then set `Command: string(cmd)`

**Comment by Crazykev** [comment]
@feiskyer move `command` marshal to server. updated.

**Comment by feiskyer** [comment]
LGTM

**Pull request merged**